models/video.js

```diff
@@ -0,0 +1,15 @@
+var database = require('./controllers/database');
+var mongoose = require('mongoose');
+var FeedItem = mongoose.model('FeedItems');
```

Comment:
🤔. weird. Mongoose must not be case sensitive. But since the DB is, I will change this for clarity.